src/components/Patient/ManagePatients.tsx

```diff
@@ -48,16 +48,18 @@ import {
   DIAGNOSES_FILTER_LABELS,
   DiagnosesFilterKey,
   FILTER_BY_DIAGNOSES_KEYS,
-} from "./DiagnosesFilter";
-import { ICD11DiagnosisModel } from "../Diagnosis/types";
-import { getDiagnosesByIds } from "../Diagnosis/utils";
-import Tabs from "@/components/Common/components/Tabs";
-import { PhoneNumberValidator } from "../Form/FieldValidators";
-import { isPatientMandatoryDataFilled } from "./Utils";
-import request from "../../Utils/request/request";
-import { Avatar } from "@/components/Common/Avatar";
+} from "./DiagnosesFilter.js";
+import { ICD11DiagnosisModel } from "../Diagnosis/types.js";
+import { getDiagnosesByIds } from "../Diagnosis/utils.js";
+import Tabs from "../Common/components/Tabs.js";
+
+import { isPatientMandatoryDataFilled } from "./Utils.js";
+import request from "../../Utils/request/request.js";
+import { Avatar } from "../Common/Avatar.js";
```

Comment:
Addition on `.js` in correct, all of those files are written in typescript. Also why this change?